setup.py

```diff
@@ -4,10 +4,23 @@
 from setuptools import setup
 from pkg_resources import parse_version
 import platform
+import sys
+import warnings
 import os
+import subprocess
 
 current_platform = platform.system().lower()
 extra_includes = []
+if current_platform == "darwin":
```

Comment:
Yeah, it seems to be working on Linux now. Another problem we now face though is that the commands module does not run in python 3.5. I might need to switch the entire pre-commit over to subprocess